framework/src/controller/application.js

```diff
@@ -84,46 +84,46 @@ class Application {
 		label,
 		genesisBlock,
 		constants = {},
-		config = { components: { logger: null } }
+		config = { components: { logger: null }, modules: {} }
 	) {
 		if (typeof label === 'function') {
```

Comment:
Not related, but can you add a comment explaining it?

Reply:
Yes

Reply:
This is refactored and resolved in other PR